client/components/share/facebook-share-preview/index.jsx

```diff
@@ -45,14 +45,18 @@ export class FacebookSharePreview extends PureComponent {
 									{ externalDisplay }
 								</a>
 								<span>
-									{ translate( 'published an article on {{a}}WordPress{{/a}}', {
+									{ translate( 'published an article on {{a}}WordPress{{/a}}.', {
 										components: {
 											a: <a href="#" />,
 										},
 									} ) }
 								</span>
 							</div>
 							<div className="facebook-share-preview__meta-line">
+								{ translate( 'Just now', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **61** times:
`translate( 'just now' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).